### PR TITLE
DAOS-10201 build: Update daos.spec to include UCX libraries, to allow…

### PR DESCRIPTION
**Pull request by jgmoore-or**
… UCX control plane code to compile in CI.

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

### CODE REVIEW

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
You need to update `Release:` and add a `%changelog` entry and a an entry to `debian/changelog` just to keep the version-release in sync between the ubuntu package and RPMs.

This PR doesn't need to also enable the UCX build in scons in some way?  `scons UCX=1` or some such?  Or should building daos default to building with UCX and it has to be specifically disabled if not wanted?

File: utils/rpms/daos.spec [hunk 1]
```diff
@@ -562,6 +562,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Wed Apr  6 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-22
+- ADD build depends entries for UCX libraries.
```
Comment (brianjmurrell) [inline utils/rpms/daos.spec:566]: If you happen to need to update this PR for anything else:

```suggestion
- Add build depends entries for UCX libraries.
```

**Review by brianjmurrell — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

File: utils/rpms/daos.spec [hunk 2]
```diff
@@ -539,6 +562,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Wed Apr  6 2022 Joseph Moore <joseph.moore@intel.com> 2.1.100-22
```
Comment (frostedcmos) [inline utils/rpms/daos.spec:565]: this should be -26, no?

Comment (brianjmurrell) [reply utils/rpms/daos.spec:565]: Good catch @frostedcmos!

**Review by brianjmurrell — CHANGES_REQUESTED** [review]
@jgmoore-or Please apply the `ADD` -> `Add` suggestion also since you have to push to fix the -22 in the `%changelog`.

File: utils/rpms/daos.spec (hunk 2 above)
Comment (johannlombardi) [reply utils/rpms/daos.spec:565]: Too late, i already landed it ...

**Pull request merged**